lib/ios/RNNAnimator.m

```diff
@@ -21,6 +21,17 @@ @interface  RNNAnimator()
 
 @implementation RNNAnimator
 
+- (instancetype)initWithAnimationsDictionary:(NSDictionary *)animationsDic {
```

Comment:
this can be easily unit-tested in native (XCtests). I'll merge the changes, plz open a separate issue on it and assign yourself